crates/unftp-sbe-gcs/src/lib.rs

```diff
@@ -58,53 +58,32 @@
 // FIXME: error mapping from GCS/hyper is minimalistic, mostly PermanentError. Do proper mapping and better reporting (temporary failures too!)
```

Comment:
I think remove this FIXME too and lets just make a ticket if anyone have an actual plan.